<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NESEmulatorControl</class>
 <widget class="QWidget" name="NESEmulatorControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>30</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="debugButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../../common/resource.qrc">
         <normaloff>:/resources/RR_bug.png</normaloff>:/resources/RR_bug.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="playButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Run</string>
       </property>
       <property name="text">
        <string>Run</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../common/resource.qrc">
         <normaloff>:/resources/stock_media-play.png</normaloff>:/resources/stock_media-play.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pauseButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Pause</string>
       </property>
       <property name="text">
        <string>Pause</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../common/resource.qrc">
         <normaloff>:/resources/stock_media-pause.png</normaloff>:/resources/stock_media-pause.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="stepCPUButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Step Into</string>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Step CPU</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../common/resource.qrc">
         <normaloff>:/resources/stepinto_6.png</normaloff>:/resources/stepinto_6.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="stepOverButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Step Over</string>
       </property>
       <property name="text">
        <string>Step CPU Over</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../common/resource.qrc">
         <normaloff>:/resources/stepover_6.png</normaloff>:/resources/stepover_6.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="stepOutButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Step Return</string>
       </property>
       <property name="text">
        <string>Step CPU Out</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../common/resource.qrc">
         <normaloff>:/resources/stepreturn_6.png</normaloff>:/resources/stepreturn_6.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="stepPPUButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Step PPU</string>
       </property>
       <property name="text">
        <string>Step PPU</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../common/resource.qrc">
         <normaloff>:/resources/continue-icon-ppu.png</normaloff>:/resources/continue-icon-ppu.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="frameAdvance">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>PPU Frame Advance</string>
       </property>
       <property name="text">
        <string>PPU Frame Advance</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../common/resource.qrc">
         <normaloff>:/resources/22_execution.png</normaloff>:/resources/22_execution.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="softButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../../common/resource.qrc">
         <normaloff>:/resources/stock_media-stop.png</normaloff>:/resources/stock_media-stop.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="resetButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Reset</string>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../common/resource.qrc">
         <normaloff>:/resources/stock_exit.png</normaloff>:/resources/stock_exit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
   <property name="toolTip">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionStep_CPU">
   <property name="text">
    <string>Step CPU</string>
   </property>
   <property name="toolTip">
    <string>Step CPU</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionStep_PPU">
   <property name="text">
    <string>Step PPU</string>
   </property>
   <property name="toolTip">
    <string>Step PPU</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Hard Reset</string>
   </property>
   <property name="toolTip">
    <string>Hard Reset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F10</string>
   </property>
  </action>
  <action name="actionFrame_Advance">
   <property name="text">
    <string>Frame Advance</string>
   </property>
   <property name="toolTip">
    <string>Advance Frame</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="actionStep_Over">
   <property name="text">
    <string>Step Over</string>
   </property>
   <property name="toolTip">
    <string>Step Over</string>
   </property>
  </action>
  <action name="actionStep_Out">
   <property name="text">
    <string>Step Out</string>
   </property>
   <property name="toolTip">
    <string>Step Out</string>
   </property>
  </action>
  <action name="actionDebugging">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Debugging</string>
   </property>
   <property name="toolTip">
    <string>Enable/Disable Debugging</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F7</string>
   </property>
  </action>
  <action name="actionSoft_Reset">
   <property name="text">
    <string>Soft Reset</string>
   </property>
   <property name="toolTip">
    <string>Soft Reset</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../../common/resource.qrc"/>
 </resources>
 <connections/>
</ui>
